laravel-app/.env.example

```diff
@@ -10,7 +10,7 @@ LOG_LEVEL=debug
 
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
-DB_PORT=3306
+DB_PORT=3307
```

Comment:
why do you need to change the .env.example??

Reply:
I used ctrl+f to find where to change po, is it not necessary na po ba to ichange ang .env.example?

Reply:
hmmm. question are you using this for your .env file??
because if so what you are doing is exposing your private credentials to everyone, you can use the .env then the .env.example is just an sample for the dev to use.

Reply:
ohhh gets po, I reverted it to the original state. I'll keep that in mind sir 🙇🙇